package.json

```diff
@@ -190,6 +190,7 @@
     "benchmark": "^2.1.4",
     "eslint": "8.57.0",
     "eslint-config-sentry-app": "2.7.0",
+    "eslint-plugin-deprecation": "^2.0.0",
```

Comment:
Do you need this in here if you are adding it in GHA step?

Reply:
oops, yeah meant to remove this and the lockfile changes.